change/@uifabric-charting-2019-09-09-15-06-50-user-v-ragor-Bug_1070386.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "minor",
```

Change type should be minor since an API is being introduced.